frontend/src/citizen-frontend/pedagogical-documents/PedagogicalDocuments.tsx

```diff
@@ -26,132 +26,132 @@ import {
 import IconButton from 'lib-components/atoms/buttons/IconButton'
 import { renderResult } from '../async-rendering'
 
-export default function PedagogicalDocuments() {
+const Attachment = React.memo(function Attachment({
+  item,
+  onRead,
+  onAttachmentUnavailable,
+  dataQa
+}: {
+  item: PedagogicalDocumentCitizen
+  onRead: (doc: PedagogicalDocumentCitizen) => void
+  onAttachmentUnavailable: () => void
+  dataQa: string
+}) {
+  return (
+    <>
+      {item && item.attachment && (
+        <FileDownloadButton
+          key={item.attachment.id}
+          file={item.attachment}
+          fileFetchFn={getAttachmentBlob}
+          afterFetch={() => onRead(item)}
+          onFileUnavailable={onAttachmentUnavailable}
+          icon
+          data-qa={dataQa}
+          openInBrowser={true}
+        />
+      )}
+    </>
```

Comment:
Unneeded fragments? 🤔 Could be just `item && item.attachment ? <FileDownloadButton ... /> : null`